src/teller/teller.go

```diff
@@ -90,17 +93,34 @@ func (s *Service) BindAddress(skyAddr string) (string, error) {
 			return "", ErrMaxBoundAddresses
```

Comment:
ErrMaxBoundAddresses needs the error message changed, because it mentions BTC

Reply:
done